### ISSUE TITLE
Can't install on Centos 6. hostname not found

### ISSUE THREAD

**Issue by pabgan** [TheLocehiliosan/yadm#122]
Hello there.

I am trying to install this as explained in [here](https://copr.fedorainfracloud.org/coprs/thelocehiliosan/yadm/):
```
 % cat /etc/centos-release                                                                                                                                                                                    
CentOS release 6.10 (Final)

  % sudo yum install yadm                                                                                                                                                                                      
Complementos cargados:fastestmirror, refresh-packagekit, security
Configurando el proceso de instalación
Loading mirror speeds from cached hostfile
 * base: mirror.linuxfix.com
 * epel: linux.mirrors.es.net
 * extras: mirror.scalabledns.com
 * updates: centos.mirror.ndchost.com
Resolviendo dependencias
--> Ejecutando prueba de transacción
---> Package yadm.noarch 0:1.12.0-1.el6 will be instalado
--> Procesando dependencias: hostname para el paquete: yadm-1.12.0-1.el6.noarch
--> Resolución de dependencias finalizada
Error: Paquete: yadm-1.12.0-1.el6.noarch (thelocehiliosan-yadm)
           Necesita: hostname
Podría intentar utilizar el comando --skip-broken para sortear el problema
Podría intentar ejecutar: rpm- Va --nofiles --nodigest

% hostname                                                                                                                                                                                                   
pganuza-dev.xxxxxx.com

% whereis hostname                                                                                                                                                                                           
hostname: /bin/hostname /usr/share/man/man7/hostname.7.gz /usr/share/man/man1/hostname.1.gz
```
Am I missing something?

Thank you for this beautiful piece of software!

**Comment by TheLocehiliosan** [comment]
Thanks for reporting this. I'ver verified this to be a dependency bug. It looks like CentOS6 is packaging `hostname` up in the "net-tools" package. I'll have to repackage for CentOS6. In the meantime, a work-around could be to do this:

    sudo yum install -y yum-utils git
    sudo rpm -Uvh --nodeps $(repoquery --location yadm)

**Comment by pabgan** [comment]
Thanks! That workaround worked. I am already using yadm.

Un saludo.
--
Pablo Ganuza Vidal

**Comment by rasa** [comment]
See fix for “hostname not found” in #124

**Comment by TheLocehiliosan** [comment]
This issues has been resolved in all RPMs hosted in OBS.
